src/Databases/Database.php

```diff
@@ -56,6 +57,7 @@ private function __construct(
         public readonly array $properties,
         public readonly DatabaseParent $parent,
         public readonly string $url,
+        public readonly bool $inline,
```

Comment:
I would rename it to `$isInline` to keep consistency across the SDK.

Reply:
Fixed!